### PR TITLE
Feat : User Profile Page

### PR DESCRIPTION
**Pull request by anuj-kumary**
Added User Profile Page

### CODE REVIEW

File: eagle-stores/src/pages/Profile/Profile.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+.container {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.profile {
+    width: 17.87rem;
+    background-color: var(--bg-color);
+    height: 21rem;
```
Comment (samyakshah3008) [inline eagle-stores/src/pages/Profile/Profile.css:16]: Can give max or min width and height for responsiveness, can refactor later on

File: eagle-stores/src/pages/Profile/Profile.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+.list__item {
+    border: none;
+    color: var(--color);
+    font-size: 1rem;
+    font-weight: 500;
+}
+
+.profile__list {
+    padding: 0.5rem 0 0 0rem;
```
Comment (samyakshah3008) [inline eagle-stores/src/pages/Profile/Profile.css:32]: Good use of shorthand property

**Review by samyakshah3008 — APPROVED** [review]
Given few small suggestions as single comment, rest is all good for approval ✅ good work!

**Review by hrishi82 — APPROVED** [review]
Code looks good to me and can be merged

File: eagle-stores/src/pages/Profile/Profile.css (hunk 1 above)
Comment (anuj-kumary) [reply eagle-stores/src/pages/Profile/Profile.css:16]: So here I am using rem units so it's already responsive

File: eagle-stores/src/App.js [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+import {
+  Home,
+  Login,
+  Signup,
+  Product,
+  Cart,
+  WishList,
+  Logout,
+  Profile,
+} from './pages';
```
Comment (PeeyushPrashant) [inline eagle-stores/src/App.js:12]: liked these barrel imports

**Review by PeeyushPrashant — APPROVED** [review]
good naming conventions
folder structure is good

**Review by Rohanmond — APPROVED** [review]
code structure looks good to merge

**Pull request merged**